docs/development.md

```diff
@@ -176,3 +177,167 @@ $ yarn run [name_command]
+`init` function of the module. That called will return a `Participation` object and you have to set that object in the 
+Redux store.
+
+
+```javascript 1.8
+
++// ----- AB Tests ----- //
+ +
+ +const participation = abTest.init();
+  
+ 
+ // ----- Redux Store ----- //
+  
+  const store = createStore(reducer);
+  
+ +store.dispatch({ type: 'SET_AB_TEST_PARTICIPATION', payload: participation });
+ 
+```
+ 
+ A real example of this you can find it [here](https://github.com/guardian/support-frontend/pull/67/files#diff-bdf2dc8b3411cc1e5f83ca22c698e7b3R41).
+
+#### Step 2: Add the AbTest reducer to your store
+
+In order to be able to understand Redux AB actions, you have to add the `abTestReducer` to your page's reducer as follows:
+ 
+```javascript 1.8
+export default combineReducers({
+  contribution,
+  intCmp,
+  abTests,
+});
+```
+  
+You can find a real example of this [here](https://github.com/guardian/support-frontend/pull/67/files#diff-c1f0bb180b22e8e0da8bde14c6b411c4R122).
+
+
+### 7.3 Implementation of a test
+
+#### Step 0: Define your experiment 
+First of all you have to **design the experiment** that you want to run. The experiment consist of a hypothesis and n 
+amount of variants. For example:
+   
+> Hypothesis: By changing [`element_to_test`], it will `[increase|decrease]` the `[put_some_metric]` by **`number`%**
+  
+The percentage of the test defined in the hypothesis impact on the length of the test. Particularly, a smaller number 
+will make the test to run for more time and a greater number will decrease the length of the test. This is related to 
+the [statistical significance concept](https://en.wikipedia.org/wiki/Statistical_significance).
+ 
+[Here](http://powerandsamplesize.com/Calculators/Compare-2-Proportions/2-Sample-Equality) you will find a tool to 
+compute the sample size of your experiment. From the sample size, you can estimate the duration of your test.  
+
+#### Step 1: Add your test to the Tests array
+
+ After your hypothesis is defined, you have to implement the test in the codebase. First, you have to define the test 
+ in the [abtest.js shared helper](/assets/helpers/abtest.js). Inside that file, under the **Tests** section you will 
+ find the definition of the tests array, you have to add a new object to this array:
+ 
+ ```javascript 1.8
+ // ----- Tests ----- //
+ 
+ const tests: Test[] = [
+   {
+     testId: 'yourTestId', 
+     variants: ['control', 'variantA'],  
+     audience: {
+       offset: 0,
+       size: 1,
+     },
+     isActive: true,
+   },
+ ];
+ ```
+  
+  Each test object has the following fields:
+  
+  * **testId**: name of the test, this name has to match with the name of the test in Abacus. Additionally, it should be 
+  unique across all the test name in Abacus. 
+  * **variants**: This field is an array of strings, each string will be a name of a variant. One of this variant name has 
```

Comment:
each string will be the **name** of a variant.  One of **these** variant **names**